packages/edit-site/src/components/header/index.js

```diff
@@ -75,6 +76,14 @@ export default function Header( {
 		setPage,
 	} = useDispatch( 'core/edit-site' );
 
+	useEffect( () => {
```

Comment:
This is a bit nitpicky, but I wonder if it would make sense to place this code closer to where the `isNavigationOpen` prop is set. Maybe in `editor/index.js`, could detect based on the `leftSidebarContent` variable

Reply:
I don't have a strong opinion about this. I'll move it 🙂